morpheus/_lib/src/objects/table_info.cpp

```diff
@@ -97,7 +102,7 @@ TableInfoData TableInfoData::get_slice(cudf::size_type start,
 
                            if (found_col == this->column_names.end())
                            {
-                               throw std::runtime_error("Unknown column: " + c);
+                               throw py::key_error("Unknown column: " + c);
```

Comment:
Shouldn't this be caught in the python wrappers and re-thrown as a key_error, or is this function only ever called by Python?

Reply:
`py::key_error` is still just a C++ error. It derives from `std::runtime_error`. The benefit here is that if its caught by pybind11, then it can be translated into a Python error. Otherwise its just a normal C++ exception.